zeq/hbp/detail/animation.fbs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2014, Human Brain Project
+//                     Cyrille Favreau <cyrille.favreau@epfl.ch>
+//
+namespace zeq.hbp;
+
+enum AmimationAction : ubyte { Pause, Play, Backwards }
+
+table Animation
+{
+  action: AmimationAction;
+  frame : uint; // frame number
```

Comment:
This is already part of frame.fbs - maybe we should simply update this event with the enum?